README.md

```diff
@@ -55,7 +55,18 @@ Just run `./gradlew integrationTest` and a container will be started and configu
 
 #### Custom Host Configuration
 
-The annotation `ContainerGebConfiguration` exists to customize the connection the container will use to access the application under test. The annotation is not required and `ContainerGebSpec` will use the default values in this annotation if it's not present.
+The annotation `ContainerGebConfiguration` exists to customize the connection the container will use to access the application under test.
+The annotation is not required and `ContainerGebSpec` will use the default values in this annotation if it's not present.
+
+The interface `IContainerGebConfiguration` exists as an inheritable version of the annotation.
+
+#### Reporting
```

Comment:
note, this comes from rebase merge-both of 5.0.x (https://github.com/grails/grails-geb/commit/b9d8663a65863cd346d457c9c8281c63ae5b10e8)
but it is ok, as this is correct:

https://github.com/grails/grails-geb/blob/7c6469eecceb266f7a93d6238f8ec0335ff3e9b4/src/testFixtures/groovy/grails/plugin/geb/GrailsGebSettings.groovy#L57